server/src/main/java/org/opensearch/index/SegmentReplicationShardStats.java

```diff
@@ -30,6 +30,7 @@ public class SegmentReplicationShardStats implements Writeable, ToXContentFragme
     private final long checkpointsBehindCount;
     private final long bytesBehindCount;
     private final long currentReplicationTimeMillis;
```

Comment:
can you pls add a comment on whats different between time & lag here? Or rename `currentReplicationTimeMillis` to `currentOngoingReplicationTimeMillis`

Reply:
Added the comment.